features/customizable-select.yml

```diff
@@ -0,0 +1,11 @@
+name: Customizable <select>
+description: The `<select>` element's appearance, including the button, selected option, picker dropdown, and options, can be customized using CSS.
+spec: https://open-ui.org/components/customizableselect/
```

Comment:
This link should be updated to https://open-ui.org/components/customizableselect/

Reply:
Isn't this the same link that's already there in the file?